notes/NewSiteArchitecture.md

```diff
@@ -0,0 +1,174 @@
+# New GraphQL.org Site Architecture:
+
+## Index
+
+*Goal:* This is the landing page and is our opportunity to quickly capture attention and explain what GraphQL is and why you should care.
+
+*Timeframe:* Launch Mon, Sept 12th
+
+This page is effectively a marketing page for GraphQL and should be the visual, scrollable version of the "Introducing GraphQL" conference talks and should be rich with visual metaphor and illustration and take advantage of whitespace to make individual salient points.
+
+Above the fold, this page should succinctly explain what GraphQL is and illustrate with a simple (editable) query/response example. Before scrolling you should understand the following:
+
+* GraphQL is an query language for APIs.
+* GraphQL is sent by client applications, such as an iOS app.
+* GraphQL is evaluated by a web service and often returned as JSON.
+* GraphQL typically replaces REST.
+
+Below the fold we should introduce concepts one at a time, each with visual metaphor and take-aways:
```

Comment:
I especially like the point about avoiding multiple endpoints (simpler).

Each of these points should be concise, visual, and quickly consumed (and all below the fold), so my concern in ordering these sections is to start with the most striking and simple to explain properties of GraphQL and then to build atop those. It's hard to explain *why* the type system is valuable without first explaining these first two points.

But if the type system is critical enough that it should be conveyed ASAP on this page, then perhaps it should be a goal for the above-the-fold to convey this point. I think that's actually quite reasonable.